src/borg/archiver.py

```diff
@@ -3967,31 +3969,35 @@ def define_borg_mount(parser):
         This command initializes an empty repository. A repository is a filesystem
         directory containing the deduplicated data from zero or more archives.
 
-        Encryption can be enabled at repository init time. It cannot be changed later.
+        Encryption mode TLDR
+        ++++++++++++++++++++
 
-        It is not recommended to work without encryption. Repository encryption protects
-        you e.g. against the case that an attacker has access to your backup repository.
+        The encryption mode can only be configured when creating a new repository -
+        you can neither configure it on a per-archive basis nor change the
+        encryption mode of an existing repository.
 
-        Borg relies on randomly generated key material and uses that for chunking, id
-        generation, encryption and authentication. The key material is encrypted using
-        the passphrase you give before it is stored on-disk.
+        Use::
 
-        You need to be careful with the key / the passphrase:
+            borg init --encryption repokey /path/to/repo
```

Comment:
well, this was intentionally not just recommended/documented like that.

`repokey` uses hmac-sha256 which is slower than `repokey-blake2` usually - except if your platform happens to have hw accelerated sha256.

so this needs an informed decision especially considering you can't change it later.